tests/framework/utils_drive.py

```diff
@@ -38,18 +38,40 @@ def partuuid_and_disk_path(rootfs_ubuntu_22, disk_path):
 
 
 VHOST_USER_SOCKET = "/vub.socket"
+CROSVM_CTR_SOCKET = "/crosvm_ctr.socket"
 
 
-def spawn_vhost_user_backend(vm, host_mem_path, socket_path, readonly=False):
+def spawn_vhost_user_backend(
+    vm, host_mem_path, socket_path, readonly=False, backend="qemu"
```

Comment:
nit: Could `backend` be an enum instead of a string?

Reply:
done